doc/UltiSnips.txt

```diff
@@ -491,7 +491,17 @@ situation. This is useful in conditional mappings.
 This function returns 1 if UltiSnips can jump backwards in the current
 situation. This is useful in conditional mappings.
 
-3.5 Warning about missing python support           *UltiSnips-python-warning*
+
+3.5 Store persistence directory                         *g:UltiSnipsStoreDir*
+-------------------------------
+
+Some interpolation data stores need persistence ('file' and 'common' sotres).
```

Comment:
typo: sotres

Reply:
Sorry for the typos, it was late when I wrote the doc, I actually just tried to convey what I had in mind the best possible and I admit I haven't reread carefuly enough

Reply:
no worries, I would not have polished at the draft state either.